vtex-apps/checkout-ui-custom
# Bugfix/south africa issue with country change

## Description: 
- This issue solves issue of `Google Maps Key` more than once
- Also this issue addresses the conflict with `defaulting` back to brazil if the `CustomAddressForm` is active and the `orderForm` brings back `FallBack` country code instead of thew current country.

## Tests:
You can test this issue [here](https://testbeto--thefoschini.myvtex.com/)

## How to test:
- Login using your account
- Add a product to the cart
- Continue checkout process
- When trying to add a `deliveryAddress` you should get `southAfrica`

## Second scenario:
- After adding the address and continuing to the `paymentMethod`, go back and edit the current address, and deliver to another address (Add another address)
